clang/test/Interpreter/anonymous-scope-fail.cpp

```diff
@@ -0,0 +1,10 @@
+// RUN: clang-repl "int x = 10;" "{ int t; a::b(t); }" "int y = 10;"
```

Comment:
Can you move this in the `code-undo.cpp` test?

Reply:
I don't quite understand, the `code-undo.cpp` doesn't seem to have anything that produces an error, but this test does.
If we define a::b(t), clang-repl stops segfaulting.

Reply:
Then we can put it in a new file called `error-recovery.cpp`.

Reply:
@p4vook ping.

Reply:
Yep, we can, I'll try to do it this evening. I couldn't figure out why this test failed though, I probably need to fix it somehow.